sdk/src/main/java/io/customer/sdk/data/store/FileStorage.kt

```diff
@@ -59,8 +59,13 @@ class FileStorage internal constructor(
 
         if (!filePath.exists()) return null
 
-        val fileContents = filePath.readText()
-        if (fileContents.isBlank()) return null
+        val fileContents = try {
+            filePath.readText()
+        } catch (ex: Exception) {
```

Comment:
Mind explaining why a try/catch is the solution to the issue?

Reply:
I now see the note in the description 😄

Reply:
😅 Added inline comment as well